Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -106,6 +107,11 @@ which you define in the instance configuration: ticket_type, query, and limit.
 For the first fetch, the integration will fetch incidents that were created 10 minutes earlier. 
 After that, the integration will fetch incidents created after the timestamp of the last fetch.
 
+Note:
+In case the **lookback** parameter is initialized with '0' and during that time incidents were fetched, if changing 
+the look back to a number that is greater > 0, then in the initial fetching there could be incidents duplications.
+If the integration was already set with look back > 0 in the first place, then those incident duplications would not occur.
```

Comment:
basically increasing the value of the parameter, even from 10 to 20 for example, will cause the same issue. let's explain this, and that it's by design but it's important to be aware.
please add this also in the `_description.md`.

Reply:
You are right, ill update the doc-string and ill update the description file as well